extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -200,7 +200,7 @@ class AmpSocialShare extends AMP.BaseElement {
       devAssert(navigator.share);
       const dataStr = href.substr(href.indexOf('?'));
       const data = parseQueryString(dataStr);
-      navigator.share(data).catch((e) => {
+      navigator.share({...data}).catch((e) => {
```

Comment:
I think it's worth adding a one line comment here :))

Reply:
This must be a bug with Chrome, it shouldn't be using the prototype for anything. Can you please file a bug?

Reply:
I updated the comment, linking to crbug.com/1123689 so we can remove this once it's fixed in WebKit.